packages/eslint-plugin/docs/rules/explicit-function-return-type.mdx

```diff
@@ -161,7 +161,7 @@ let funcExpr: FuncType = function () {
 };
 
 let asTyped = (() => '') as () => string;
-let castTyped = <() => string>(() => '');
+let assertTyped = <() => string>(() => '');
```

Comment:
I've tried to somewhat match the wording on the `asTyped` and `objectPropAs` variable names. Since the AST-node name for angular bracket assertion is `TSTypeAssertion`, I think it makes sense to use the assert wording here.

Reply:
Honestly, what if we just remove those examples? 🤔 Angle-bracket type assertions are yucky, and we don't really want to encourage them. Even with the name "castTyped"/"assertTyped" it took me longer than I care to admit to make sense of those examples 😆

Reply:
Yeah, I was confused by some of these examples too. I don't think the additional angle bracket assertion adds any value, either.

I'll be happy to remove them. Should I push it as part of this PR?

Reply:
Done 👍